### ISSUE TITLE
How to set search parameters

### ISSUE THREAD

**Issue by richtong** [themefisher/parsa-hugo#39]
This theme is *awesome* really nice work, I did fork it and added some documentation (do you want the PR on it like what does layout 1 or 2 mean).

But one thing that I broke but can't figure out is that Search no longer works, if you go to https://restart.us which is a fork of this, the search is using an absolute path, so it tries to do a `https://search.js?q=foo` instead of `https://restart.us/search.js?q=foo` do you know where the template is that specifies this?

I'm a complete noob, but the search does a correct URL when running in test mode so with `hugo server`, but when I push it to netlify, it doesn't, so something is different up there. I'm having trouble seeing how in Hugo, you pass the parameter to the Search page, but I presume there is some documentation somewhere about search.js and how to configure the link properly.

**Comment by somratpro** [comment]
Hello @richtong 

We just update the theme. Please download the latest code. It will solve your issue I guess.

**Comment by richtong** [comment]
Got it, ok, I did a `hugo mod vendor` to pull the latest, but it doesn't seem to fix it, what's the right way to do this? See https://responsible.org

**Comment by richtong** [comment]
Oh I see you should add to your documentation

To update to latest theme:

```
hugo mod get -u ./...
hugo mod vendor
```

Which will pull all the modules into the new repo

**Comment by richtong** [comment]
Ok I think I have it all running now on https://responsible.org. 

The README is pretty confusing how to do this. Would like a PR to the README that explains how to do it. 

Overall moving to the Hugo module method is much cleaner FYI than using themes with a sub module. 

We had to overlay the homepage as an example. 

Also you've removed the menu items for about and contact so it's jot as usiful as a out of the box site. I just added these back. 

Finally the love to think from yaml to toml threw me off but got all that running too. So thanks. 

Overall these are great templates but confusing for a noob
